keras/engine/topology.py

```diff
@@ -1229,6 +1229,45 @@ def count_params(self):
                                    self.name + '.build(batch_input_shape)`.')
         return sum([K.count_params(p) for p in self.weights])
 
+    def count_flops(self):
```

Comment:
`flops` is generally used for floating point operations per second. Since this is merely a flop count, maybe we need a different name. Maybe.